markdown/events/2023/bytesize_mag.md

```diff
@@ -21,3 +24,89 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[8:33](https://www.youtube.com/watch?v=IiorfDHeoLo&t=513)
+And with this, I've shown you the rough overview of the pipeline and next I would like to show you some of the impact, different assembly settings can have. For this I simulated some mouse gut data set in the past with the tool CAMISIM, and I generated hybrid data containing Illumina data and Nanopore reads and generated two groups. Each with a time series of four samples. This might be the ideal case where a core assembly might be useful. Now I would like to show you some of the resulting assembly metrics that are commonly used.
+
+[9:14](https://www.youtube.com/watch?v=IiorfDHeoLo&t=554)
+Here you can see for example the total length of the resulting assemblies and then compared for different pipeline ones were different assembly settings were used. The lower two pipeline settings correspond to a sample-wise assembly and using either only short or short and long read, so hybrid data, and the upper two settings correspond to a core assembly. Again with short or short and long reads, and what we can see is that the total length of the resulting assemblies significantly increased both by using the hybrid setting, and by applying the core assembly setting. Similar results we also see when looking at the number of MAGs, so the number of genomes that could be retrieved from this data, and also when looking at the L50 values. This indicates that the actual setting that is used for the assembly within this pipeline can have a relatively huge impact on the results. It's definitely good that the pipeline provides different settings, so that you can really choose the correct setting for input data, and it might also be worth to compare different settings.
+
+[10:32](https://www.youtube.com/watch?v=IiorfDHeoLo&t=632)
+Another topic I would like to shortly mention is the resource requirements, because this came up quite often in the Slack channel, and it's also somehow difficult to estimate in advance, because it really differs depending on the input data. The main requirements are both for memory and time, coming from the assembly step. As I mentioned already it really differs for different input data sets and I collected some numbers just to give you a rough idea for different pipeline ones that were run by Daniel Straub on our compute cluster. For one rather small sample, which was a cultural sample, both MegaHIT and SPAdes required less than 25 gigabytes and were finished in a couple of hours. However, for a larger river sample data set, MegaHIT took already more than 100 gigabytes of RAM, and it took more than one day to finish, and SPAdes even took more than 900 gigabytes of memory, and it required more than nine days. There was another very large data set containing 15 soil samples for which also a core assembly was performed and for this MegaHIT required one terabyte and more than 17 days, and SPAdes could not even be run because it would have required more than two terabytes of memory.
```

Comment:
```suggestion
Another topic I would like to shortly mention is the resource requirements, because this came up quite often in the Slack channel, and it's also somehow difficult to estimate in advance, because it really differs depending on the input data. The main requirements are both for memory and time, coming from the assembly step. As I mentioned already it really differs for different input data sets and I collected some numbers just to give you a rough idea for different pipeline runs that were run by Daniel Straub on our compute cluster. For one rather small sample, which was a culture sample, both MegaHIT and SPAdes required less than 25 GB and were finished in a couple of hours. However, for a larger river sample data set, MegaHIT took already more than 100 GB of RAM, and it took more than one day to finish, and SPAdes even took more than 900 GB of memory, and it required more than nine days. There was another very large data set containing 15 soil samples for which also a co-assembly was performed and for this MegaHIT required one TB and more than 17 days, and SPAdes could not even be run because it would have required more than two TB of memory.
```